README.md

```diff
@@ -17,9 +17,40 @@ A few key features:
 
 ## Getting Up and Running
 
-[Using a docker container](docs/docker_container.md)
+Start using OpenVino&trade; Model Server in 5 Minutes or less
 
-[Landing on bare metal or virtual machine](docs/host.md)
+```bash
+# Download the OpenVino Model Server image
+docker pull openvino/ubuntu18_model_server:latest
+
+# Download model into a separate directory
+curl --create-dirs https://download.01.org/opencv/2020/openvinotoolkit/2020.2/open_model_zoo/models_bin/3/face-detection-retail-0004/FP32/face-detection-retail-0004.xml https://download.01.org/opencv/2020/openvinotoolkit/2020.2/open_model_zoo/models_bin/3/face-detection-retail-0004/FP32/face-detection-retail-0004.bin -o model/face-detection-retail-0004.xml -o model/face-detection-retail-0004.bin
+
+# Start the Model Server container serving the gRpc port
```

Comment:
Change to `# Start the Model Server container serving gRPC on port 9000

Reply:
done